src/pages/order-item/tabs/delivery-details/courier/courier.js

```diff
@@ -26,17 +31,24 @@ const Courier = ({ deliveryType, values, handleChange }) => {
           {deliveryTitles.deliveryAddress}
         </h4>
         <div>
-          {courierInputLabels().map((field) => (
-            <TextField
-              key={`${field.name}${deliveryType}`}
-              name={field.name}
-              data-testid={field.name}
-              className={styles.textField}
-              label={field.label}
-              value={values[field.value]}
-              onChange={handleChange}
-              variant={materialUiConstants.outlined}
-            />
+          {courierInputLabels().map((field, idx) => (
+            <React.Fragment key={idx}>
+              <TextField
```

Comment:
You can add the error prop to all textfields you worked with, so errors will be more noticeable

Reply:
will do, thanks

Reply:
added the error prop